### DISCUSSION TITLE
[预览] v2.8.3

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#4386] [分享 / Share]
包含 v2.7.5 的所有更新内容.

✨新增
- `网址参数清理` 更新对直播间的支持. (#1459)
- `自动点赞` 转为由 [CrazyboyQCD](https://github.com/CrazyboyQCD) 维护, 支持手动控制点赞和黑名单功能. (PR #4343, PR #4358 by [CrazyboyQCD](https://github.com/CrazyboyQCD))

- `简化直播间` 支持隐藏荣耀等级勋章. (PR #4348 by [CrunchyShark](https://github.com/CrunchyShark944))
- 在设置面板中的组件管理等面板中, 搜索框移动至已安装列表的上方, 更符合直觉. (#3806)
- 支持从 Tampermonkey 的菜单中唤起功能和设置面板. (#4170)
- 搜索栏中添加了导入 / 导出设置的操作. (#4170)
- `自定义顶栏` 新增选项 `链接对齐样式`, 可以自定义纯链接弹窗内链接文字的对齐样式. (PR #4365 by [Tinhone](https://github.com/Tinhone))
- `简化评论区` 支持独立控制简化选项, 并支持隐藏粉丝勋章和活动横幅. (仅支持新版评论区) (#2381)
- `清爽首页` 设置为隐藏的版块现在将彻底销毁, 减少资源占用.
- `禁用特殊弹幕样式` 支持禁用大会员弹幕. (#4227)
- `下载视频` 在批量下载番剧时, 支持下载多个不同板块的选集. (#2834)
- 新增功能 `全屏直播礼物简化`. (PR #4306 by [TimmyOVO](https://github.com/TimmyOVO))

> 移除全屏观看直播时的底部礼物栏

- 新增功能 `评论区 IP 属地显示`. (PR #4331, PR #4334 by [Light_Quanta](https://github.com/LightQuanta))

> 在评论区显示评论的IP属地信息

- 新增功能 `直播间网页全屏自适应`. (#4216)

> 在直播网页全屏时, 自动调整侧边栏的宽度, 使得视频区域的比例和视频源相匹配, 达到无黑边的效果.
>
> - `侧边栏最大宽度 (px)`: 限制侧边栏可被拉伸到的最大宽度. (最小宽度固定为 190px, 再小的话布局就要出问题了)
>
> > 注意, 由于有最大宽度和最小宽度的限制, 部分窗口尺寸下仍然无法做到无黑边.

- 新增功能 `关注时间显示`. (PR #4352 by [Light_Quanta](https://github.com/LightQuanta))

> 在个人空间的粉丝/关注列表显示关注的具体时间

- 新增功能 `自动移出稍后再看`.

> 在稍后再看页面播放结束时, 自动将当前视频移出稍后再看.
> 注意:
> - 一定要播放结束, 快结束时手动切走不算
> - b 站的稍后再看列表不会实时刷新

- 新增功能 `相簿发布时间显示`. (PR #4362 by [Light_Quanta](https://github.com/LightQuanta))

> 在个人空间的相簿界面显示相簿的发布时间

🐛修复
- 修复 `清爽首页` 番剧区图标不显示. (#4262)
- 修复 `清爽首页` 番剧区时间线的遮罩颜色在夜间模式下不正确.
- 修复 `隐藏记笔记` 功能打开后, 评论区的笔记弹窗无法显示. (#4285)

☕开发者相关
- SwitchOptions API 更新:
  - 老的 `createSwitchOptions` 标记为已弃用.
  - 新的 `newSwitchComponentWrapper` 更名为 `wrapSwitchOptions`.
  - `checkedIcon` 拥有默认值: `mdi-eye-off-outline`.
  - `notCheckedIcon` 拥有默认值: `mdi-eye-outline`.
  - `dimAt` 选项功能更改为如下所示, 更符合直觉:
```ts
/**
 * 控制开关变暗的时机
 *
 * `false`: 始终不变暗
 * `'checked'`: 当开关开启时变暗
 * `'notChecked'`: 当开关关闭时变暗
 *
 * @default 'checked'
 */
dimAt?: false | 'checked' | 'notChecked'
```
- Vue 升级到 2.7, 支持组合式 API 以及使用 Volar 提供代码提示. (PR #4337 by [timongh](https://github.com/timongh))
- 修复组件加载失败时没有输出任何报错.

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.8.3-preview'>[预览] v2.8.3</a>.</em>

**Comment by yongrenx** [comment]
emmm不知道为什么升级了却没有生效

**Reply by Tinhone** [reply]
有可能是因为脚本在下载 lodash 库，需要等一会才能启用

**Reply by yongrenx** [reply]
还是没办法，删除脚本重装也不行，甚至设置还是原来的，这是什么bug

**Reply by Geeyun-JY3** [reply]
> 还是没办法，删除脚本重装也不行，甚至设置还是原来的，这是什么bug

啥叫「设置还是原来的」，你是不是开了 Tampermonkey 的回收站功能？

**Reply by yongrenx** [reply]
嗯，已经发现了，删除后重装能解决问题，但是导入原来设置后新增功能就又没了，需要重新安装那个功能

**Comment by theblueandblackcrow** [comment]
更新了新的版本后，发现番剧方面的问题（API问题）还没有修好
是在下一版本进行修复吗？🤔

**Reply by Geeyun-JY3** [reply]
这个？：https://github.com/users/the1812/projects/1/views/3?pane=issue&itemId=36866403
https://github.com/the1812/Bilibili-Evolved/issues/4305#issuecomment-1691940565

**Reply by theblueandblackcrow** [reply]
这个我也在下面留言了
但是问题还在，可能是api的调整不太好搞吧

**Reply by Tinhone** [reply]
https://github.com/users/the1812/projects/1/views/3

Projects 里面写了相关的计划，下一个版本应该会修复番剧 API 的问题

**Reply by theblueandblackcrow** [reply]
我看到了
感谢提供指引
